plugins/search-react/src/components/SearchPagination/SearchPagination.tsx

```diff
@@ -176,11 +188,13 @@ export type SearchPaginationProps = Omit<
  * @public
  */
 export const SearchPagination = (props: SearchPaginationProps) => {
-  const { pageLimit, setPageLimit, pageCursor, setPageCursor } = useSearch();
+  const { pageLimit, setPageLimit, pageCursor, setPageCursor, fetchNextPage } =
```

Comment:
looks like there is a [`hasNextPage`](https://github.com/backstage/backstage/blob/master/plugins/search-react/src/context/SearchContext.tsx#L139) on the search context already, do you think it make sense to return that and use that here instead of `fetchNextPage` ?

Reply:
hi @emmaindal ! I looked at the search context and it looks like `hasNextPage` is an internal variable that's not actually exposed as part of the [search context](https://github.com/backstage/backstage/blob/master/plugins/search-react/src/context/SearchContext.tsx#L42-L63) which is why I elected to use `fetchNextPage` as a ["proxy"](https://github.com/backstage/backstage/blob/master/plugins/search-react/src/context/SearchContext.tsx#L180) for `hasNextPage`. 

If you think it makes more sense to update `SearchContext` to expose `hasNext`/`hasPreviousPage`, I can make that change instead?

Reply:
yeah its not exposed as of now and maybe because there has been no need for it, but I didn't see much reasons to not to do that. But interested if you think it should be kept internal?